docs/.vuepress/public/schemas/Prices.schema.json

```diff
@@ -0,0 +1,7 @@
+{
+  "paper": {
+    "type": "object(string: float)",
+    "example": "\"2019-04-28\": 0.25",
+    "description": "Paper format prices of a card."
```

Comment:
Please add "Courtesy of [mtgstocks](https://www.mtgstocks.com?utm_campaign=mtgjson&utm_medium=mtgjson&utm_source=mtgjson)." After the description.